mongodb/kubedb_client_builder.go

```diff
@@ -122,6 +122,11 @@ func (o *KubeDBClientBuilder) GetMongoClient() (*Client, error) {
 
 func (o *KubeDBClientBuilder) getURL() string {
 	nodeType := o.podName[:strings.LastIndex(o.podName, "-")]
+	if strings.Contains(nodeType, api.NodeTypeArbiter) {
```

Comment:
What if the db name has arbiter in it? ex: `arbiter-mongodb`.

Use `strings.HasSuffix()`.